tools/XRefService/DatabaseLib/Utility/MD5Encrytion.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Security.Cryptography;
+using System.Text;
+
+namespace DatabaseLib.Utility
+{
+    public class MD5Encryption
+    {
+        public static string CalculateMD5Hash(string input)
+        {
+            // step 1, calculate MD5 hash from input
+            MD5 md5 = MD5.Create();
+            byte[] inputBytes = Encoding.UTF8.GetBytes(input);
```

Comment:
it's not a namespace it's a class

---
In reply to: [132074621](https://github.com/dotnet/docfx/pull/1906#discussion_r132074621) [](ancestors = 132074621)